### PR TITLE
Remove usage of gene.info from pipeline

### PR DESCRIPTION
**Pull request by bramalingam**
Tag: EMBLPredoc2017Training

This version is specific for EMBLPredoc2017Training. Will need a copy of gene info database locally for getting entrez ids.

@jburel @joshmoore @manics

### CODE REVIEW

**Comment by jburel** [comment]
We will probably need to use a flag to enable it otherwise if we need the entries for something else we will need a new tag each time
We can create an issue and look into it later on

**Comment by jburel** [comment]
``
idr/attributes.py:8:1: F401 'externalDBs.get_entrezid' imported but unused
``

File: idr/attributes.py [hunk 1]
```diff
@@ -196,7 +196,8 @@ def get_phenotypes_for_gene(session,
 def get_phenotypes_for_genelist(session,
                                 go_gene_list,
                                 organism,
-                                idr_base_url="https://idr.openmicroscopy.org"):
+                                idr_base_url="https://idr.openmicroscopy.org",
+                                entrez_flag=True):
```
Comment (manics) [inline idr/attributes.py]: can you make this more descriptive, e.g. `lookup_entrez`?

File: idr/attributes.py [hunk 2]
```diff
@@ -240,13 +243,14 @@ def get_phenotypes_for_genelist(session,
...
         # search with entrezid if gene symbol and
         # ensembleid does not return any result
-        # if len(uniquelist) == 0:
-        #     key = "EntrezID"
-        #     for gid in entrezid:
-        #         uniquelist = get_phenotypes_for_gene(session,
-        #                                              gid)
-        #         if len(uniquelist['Name']) != 0:
-        #             break
+        if entrez_flag == True:
```
Comment (manics) [inline idr/attributes.py]: Just use `if entrez_flag:`, you don't need `== True`

**Pull request merged**